physics/module_mp_thompson.F90

```diff
@@ -1,6545 +0,0 @@
-!>\file module_mp_thompson.F90
```

Comment:
Were any changes made to this file when it was moved? Since there isn't an associated PR for the new submodule, it's impossible to tell/review any changes that you may have put in the moved file.